### PR TITLE
Add plausible tracking

### PR DESCRIPTION
**Pull request by lizokm**

### CODE REVIEW

File: docs/concepts/migration/index.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "Learn more about the reasons to move to Sentry's SaaS solution, wh
 Sentry offers a cloud-hosted, software-as-a-service (SaaS) solution in addition to a self-hosted solution, which are both functionally the same. However, many customers find that self-hosted Sentry can quickly become expensive to maintain, scale, and support, making our SaaS product the better and less costly option. To facilitate moving from self-hosted to SaaS, we provide a self-serve process known as "relocation".
 
 <Alert>
-Check out this video on [**Migrating to Sentry SaaS**](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to learn about our relocation tooling.
+Check out this video on [**Migrating to Sentry SaaS**](<a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">Migrating to Sentry SaaS</a>) to learn about our relocation tooling.
```
[the remark this answers is not in the document]
Comment (lizokm) [reply docs/concepts/migration/index.mdx:10]: @whitep4nth3r apparently this is all you need :)

**Comment by coolguyzone** [comment]
Hi @lizokm , I just checked this out on https://sentry-docs-git-add-plasuible-tracking-link-to-video.sentry.dev/concepts/migration/ and am seeing the following:

Screenshot 2024-10-30 at 1 48 22 PM

Looks like it's not quite right yet

**Review by coolguyzone — APPROVED** [review]
Good to go!

**Comment by lizokm** [comment]
@coolguyzone thank you for catching that! That comment was supposed to go on a different PR!

File: docs/concepts/migration/index.mdx (hunk 1 above)
Comment (lizokm) [inline docs/concepts/migration/index.mdx:10]:
```suggestion
Check out this video on (<a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">**Migrating to Sentry SaaS**</a>) to learn about our relocation tooling.
```

File: docs/concepts/migration/index.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: "Learn more about the reasons to move to Sentry's SaaS solution, wh
 Sentry offers a cloud-hosted, software-as-a-service (SaaS) solution in addition to a self-hosted solution, which are both functionally the same. However, many customers find that self-hosted Sentry can quickly become expensive to maintain, scale, and support, making our SaaS product the better and less costly option. To facilitate moving from self-hosted to SaaS, we provide a self-serve process known as "relocation".
 
 <Alert>
-Check out this video on [**Migrating to Sentry SaaS**](https://sentry.io/resources/migrate-to-sentry-saas-workshop/) to learn about our relocation tooling.
+Check out this video on (<a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">**Migrating to Sentry SaaS**</a>) to learn about our relocation tooling.
```
Comment (coolguyzone) [inline docs/concepts/migration/index.mdx:10]:
```suggestion
Check out this video on <a href="https://sentry.io/resources/migrate-to-sentry-saas-workshop/" className="plausible-event-name=migrating+to+saas+video">**Migrating to Sentry SaaS**</a> to learn about our relocation tooling.
```

I'm seeing the parentheses around the link on https://sentry-docs-git-add-plasuible-tracking-link-to-video.sentry.dev/concepts/migration/

**Pull request merged**